cypress/integration/spawn/volume.ts

```diff
@@ -184,26 +184,37 @@ describe("Navigating to Spawn Volume page", () => {
   });
 
   describe("Migrate Modal", () => {
-    it("open the Migrate modal and spawn a host", () => {
+    const cueCopy = "You can now migrate your home volume to a new spawn host!";
```

Comment:
In this case I think it's better to use `data-cy` then, since the dialog role is too ambiguous. I'm not really a huge fan of looking at the text to check that the guide cue is gone. I checked and it looks like `data-cy` will properly be applied to the element:

<img width="425" alt="Screen Shot 2022-11-04 at 3 38 50 PM" src="https://user-images.githubusercontent.com/47064971/200061103-5b7e64f3-c56a-4d56-8e7b-af5ec64df022.png">